salt/modules/win_task.py

```diff
@@ -1382,14 +1382,16 @@ def info(name, location='\\'):
             trigger['end_date'] = end_date
             trigger['end_time'] = end_time
         trigger['enabled'] = triggerObj.Enabled
-        if triggerObj.RandomDelay:
-            trigger['random_delay'] = _reverse_lookup(duration, triggerObj.RandomDelay)
-        else:
-            trigger['random_delay'] = False
-        if triggerObj.Delay:
-            trigger['delay'] = _reverse_lookup(duration, triggerObj.Delay)
-        else:
-            trigger['delay'] = False
+        if hasattr(triggerObj, 'RandomDelay'):
+            if triggerObj.RandomDelay:
```

Comment:
Could this be combined into a single if statement using `getattr`:
```
if getattr(triggerObj, 'RandomDelay'):
    trigger['random_delay'] = _reverse_lookup(duration, triggerObj.RandomDelay)
else:
    trigger['random_delay'] = False
```

Reply:
Hi twangboy, I'm not sure it is a good idea because some trigger type doesn't have a RandomDelay or a Delay. Ex.: The LOGON trigger type has a Delay attribute but no RandomDelay attribute. So, showing False for an attribute that doesn't exist is strange I think.